setup.py

```diff
@@ -9,9 +9,9 @@
   license="MIT",
   include_package_data=True,
   py_modules=["instiki2git"],
-  install_requires=["PyMySQL", "dulwich", "click", "requests"],
+  install_requires=["PyMySQL", "dulwich", "click", "requests", "configparser"],
   entry_points="""
     [console_scripts]
     instiki2git=instiki2git:cli
     instiki2git-html=instiki2git:cli_html
-  """)
\ No newline at end of file
+  """)
```

Comment:
Consider adding [python_requires](https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#python-requires) ~= [3.9](https://peps.python.org/pep-0440/#compatible-release)

Reply:
Did you see this? Feel free to "resolve the conversation".

Reply:
I saw it. I just struggled a bit with finding documentation...

Reply:
I only used what I linked.